test/views/components/description_list/DescriptionListSpec.scala

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.components.description_list
+
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import org.jsoup.select.Elements
+import org.scalatest.matchers.must.Matchers.convertToAnyMustWrapper
+import play.api.Application
+import play.api.i18n.Messages
+import play.twirl.api.Html
+import utils.SpecBase
+import views.html.components.description_list.{dd, dl, dt}
+
+class DescriptionListSpec extends SpecBase {
+
+  "dd component" should {
+
+    "display correct contents" when {
+
+      "only content has been provided" in new Setup {
+        val dd: Elements = ddComponent.select("dd")
```

Comment:
Please add a line break after this line to differentiate between result and assertions. Same for other tests

Reply:
This has been addressed in the commit [73dfe78](https://github.com/hmrc/customs-financials-documents-frontend/pull/101/commits/73dfe781532fd8bddb0967c9eeb15974440beaf7)